### DISCUSSION TITLE
Revise use of remirror social editor

### DISCUSSION THREAD

**Discussion by mtfoley** [open-sauced/open-sauced#1360] [Ideas]
I glanced at this article that @blackgirlbytes co-authored: https://www.stackbit.com/blog/open-source-rich-text-editor/

our implementation of remirror was very shortly after their 1.0 and I think a cogent version of the social had not quite landed. 

Just wanted to raise a discussion of whether this was worth looking into. 

PS: I think there are unreported state management bugs in the notes editor. Try opening and canceling, the note is blank, IRC.

cc @ifiokjr

**Comment by blackgirlbytes** [comment]
I can try out the notes editor tomorrow and come back with more thoughts, but if we decide on implementing it..happy to help out!

**Reply by mtfoley** [reply]
Cool. IMHO, the opportunities for improvement on it are:
- acting as a "controlled" component. Right now we update state on every change instead of on save.
- finding a more efficient way to query and render emojis. At the moment the emoji list is a 2MB portion of our bundle and the images are lazy loaded in a 3MB sprite.
